docs/howto/catalog-queries.md

```diff
@@ -12,61 +12,51 @@ curl -k https://localhost:8443/catalogs/operatorhubio/api/v1/all | <query>
 
 ## Package queries
 
-Available packages in a catalog
-: 
```

Comment:
This formatting creates a definition list. Please see: https://squidfunk.github.io/mkdocs-material/reference/lists/#using-definition-lists.

You can see the difference in formatting between the live docs and the rendered preview:
1. [Published docs](https://operator-framework.github.io/operator-controller/howto/catalog-queries/#package-queries)
2. [Preview docs](https://deploy-preview-1443--olmv1.netlify.app/howto/catalog-queries/#package-queries)

Reply:
Ahh, good to know! Didn't think it was intentional.